fml/platform/android/jni_util.cc

```diff
@@ -25,7 +26,18 @@ JNIEnv* AttachCurrentThread() {
   FML_DCHECK(g_jvm != nullptr)
       << "Trying to attach to current thread without calling InitJavaVM first.";
   JNIEnv* env = nullptr;
-  jint ret = g_jvm->AttachCurrentThread(&env, nullptr);
+  JavaVMAttachArgs args;
+  args.version = JNI_VERSION_1_4;
+  args.group = nullptr;
+  // 16 is the maximum size for thread names on Android.
+  char thread_name[16];
+  int err = prctl(PR_GET_NAME, thread_name);
+  if (err < 0) {
```

Comment:
nit: should probably be `err != 0` or some enum if one exists like `err != kResultOk`